src/core/qgstemporalnavigationobject.h

```diff
@@ -255,6 +281,8 @@ class CORE_EXPORT QgsTemporalNavigationObject : public QgsTemporalController, pu
     //! The controller temporal navigation extent range.
     QgsDateTimeRange mTemporalExtents;
 
+    NavigationMode mNavigationMode = Animated;
```

Comment:
I think we should default to off here

Reply:
We could -- I was aiming at default behavior parity, open to having it off by default.

Reply:
Yeah - but it's bothered me a little that we default to a temporal filter immediately on adding a temporal source. It could potentially be confusing to users who would expect to see all the features/layer content show up immediately on adding a new layer

Reply:
@nyalldawson , OK, let's change the default mode.

Reply:
as with off by default, mesh layer could be not visible, I purpose this PR #36494